docs/src/user-guide/command-line-interface.md

```diff
@@ -370,83 +469,84 @@ npx eslint --quiet file.js
 
 This option allows you to specify a warning threshold, which can be used to force ESLint to exit with an error status if there are too many warning-level rule violations in your project.
 
-Normally, if ESLint runs and finds no errors (only warnings), it exits with a success exit status. However, if `--max-warnings` is specified and the total warning count is greater than the specified threshold, ESLint exits with an error status. To prevent this behavior, omit this option or specify a threshold of `-1`.
+* **Argument Type**: Integer. The maximum number of warnings to allow. To prevent this behavior, do not use this option or specify `-1` as the argument.
+* **Multiple Arguments**: No
 
-Example:
+Normally, if ESLint runs and finds no errors (only warnings), it exits with a success exit status. However, if `--max-warnings` is specified and the total warning count is greater than the specified threshold, ESLint exits with an error status.
+
+##### `--max-warnings` example
 
 ```shell
 npx eslint --max-warnings 10 file.js
 ```
 
 ### Output
 
-#### `-o`, `--output-file`
+#### `--output-file`, `-o`
+
+Write the output of linting results to a specified file.
 
-Enable report to be written to a file.
+* **Argument Type**: String. Path to file.
+* **Multiple Arguments**: No
 
-Example:
+##### `--output-file`, `-o` example
 
 ```shell
 npx eslint -o ./test/test.html
 ```
 
-When specified, the given format is output into the provided file name.
+#### `--format`, `-f`
 
-#### `-f`, `--format`
+This option specifies the output format for the console.
 
-This option specifies the output format for the console. Possible formats are:
+* **Argument Type**: String. One of the [built-in formatters](formatters/) or a custom formatter.
+**Multiple Arguments**: No
```

Comment:
```suggestion
* **Multiple Arguments**: No
```

Missing `*` for list item.

![image](https://user-images.githubusercontent.com/44349756/204539938-99faa9a4-b10b-4347-afda-861d1c0bc6bd.png)